.github/workflows/blackbox-pr.yml

```diff
@@ -9,5 +9,7 @@ on:
 jobs:
   blackbox-tests:
     name: Blackbox Tests
-    if: contains(github.event.pull_request.labels.*.name, 'Run Blackbox')
+    if:
```

Comment:
Ensure regular blackbox tests do not run when the `Run Blackbox (LTS)` is the only tag present